src/nvim/eval/typval.h

```diff
@@ -426,6 +426,18 @@ static inline void tv_list_ref(list_T *const l)
   l->lv_refcount++;
 }
 
+/// Set a list as the return value
+///
+/// @param[out]  tv  Object to receive the list
+/// @param[in,out]  l  List to pass to the object
+static inline void tv_list_set_ret(typval_T *const tv, list_T *const l)
+  REAL_FATTR_NONNULL_ARG(1)
```

Comment:
Not correct again, may not compile at all with some attributes (including rather common in such cases REAL_FATTR_ALWAYS_INLINE AFAIR). You need to have declaration separately and attach attribute to *that*. Check just the next function, it has declaration with attributes right before definition. Declaration may be omitted if you do not need any attributes attached.